src/opentype/tables/advanced/lookups/gsub/lookup-type-7.js

```diff
@@ -1,12 +1,48 @@
-import { ParsedData } from "../../../../../parser.js";
+import { LookupType, undoCoverageOffsetParsing } from "./gsub-lookup.js";
 
-class LookupType7 extends ParsedData {
-  // note: not "extends LookupType"
+import { LookupType1 } from "./lookup-type-1.js";
+import { LookupType2 } from "./lookup-type-2.js";
+import { LookupType3 } from "./lookup-type-3.js";
+import { LookupType4 } from "./lookup-type-4.js";
+import { LookupType5 } from "./lookup-type-5.js";
+import { LookupType6 } from "./lookup-type-6.js";
+import { LookupType8 } from "./lookup-type-8.js";
+
+// This subtable is the "I actually need a 32 bit
+// offset" hack table that is used as a pointer to
+// any of the other subtables that can't be pointed
+// to using a 16 bit offset value.
+class LookupType7 extends LookupType {
```

Comment:
It doesn't seem we need much from LookupType, and we remove/overwrite half of it. We extended `ParsedData` before, can't we keep doing that?

- We delete `coverageOffset`
- We delete `getCoverageTable`
- Since we're not calling `getCoverageTable` we don't need to move back two bytes
- We do need `substFormat` but we can look it up ourselves in the `LookUpType7`

I feel like I've only _just_ wrapped my head around the binary parsing so I might miss a thing or two. But what do you think?

Reply:
It probably makes more sense to treat it like a real subtable and parse it as such, just to keep the code structured roughly the same across the various files.

Reply:
But I *am* going to compact that type list a little, that can just be an array...